### PR TITLE
Update broken link to VS

### PR DESCRIPTION
**Pull request by omidkrad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ In order to be able to get new builds of the DNX, and switch between them, you n
 
 ## Getting Started on Windows
 
-The easiest way to get started on Windows is to grab the latest preview of Visual Studio 2015, which can be found [here](http://go.microsoft.com/fwlink/?LinkId=521794).
+The easiest way to get started on Windows is to grab Visual Studio 2015 Community, which can be found [here](https://www.visualstudio.com/en-us/products/visual-studio-community-vs.aspx).
```
Comment (Eilon) [inline README.md]: There should probably be links to both editions, right?

Comment (omidkrad) [inline README.md]: Code and Community?

Comment (Eilon) [inline README.md]: Oh I was thinking Enterprise and Community. But linking to VS Code ain't a bad idea either.

@DamianEdwards @danroth27 any thoughts on this one?

Comment (danroth27) [inline README.md]: We are maintaining our official getting started instructions for Windows at http://docs.asp.net/en/latest/getting-started/installing-on-windows.html. On Windows I prefer to keep the focus full VS.

Comment (glennc) [inline README.md]: So community and enterprise then? Code doesn't really help with getting started as you need to get the yeoman generators and such. So I am ok with that.

**Comment by Eilon** [comment]
Closing because this is no longer needed. Thanks!

**Pull request closed without merging**